examples/inspection/plot_partial_dependence.py

```diff
@@ -1,74 +1,82 @@
 """
-========================
-Partial Dependence Plots
-========================
+===============================================================
+Partial Dependence and Individual Conditional Expectation Plots
+===============================================================
 
 Partial dependence plots show the dependence between the target function [2]_
 and a set of 'target' features, marginalizing over the values of all other
 features (the complement features). Due to the limits of human perception, the
-size of the target feature set must be small (usually, one or two) thus the
+size of the 'target' feature set must be small (usually, one or two) thus the
 target features are usually chosen among the most important features.
 
-This example shows how to obtain partial dependence plots from a
+Similarly, an individual conditional expectation (ICE) plot [3]_
+shows the dependence between the target function and a 'target' feature.
```

Comment:
Good idea. Then it would be consistent with docs (https://github.com/scikit-learn/scikit-learn/pull/16619/files#diff-a796d02ac30378a0519e2b96771b1f79)

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/58ddd8220bda876acbcf69e87eed98550474dbe2